mirrord-agent/src/file.rs

```diff
@@ -0,0 +1,113 @@
+use std::{
```

Comment:
P.S all the file ops need to run in the namespace of the container id given to us, as sniffer does. Right now you probably relay the calls to the pod host rather then to the pod ;)